Buildings/Controls/OBC/ASHRAE/G36_PR1/Generic/SetPoints/OperationMode.mo

```diff
@@ -306,6 +308,22 @@ protected
     "Generate alarm message when window open during modes other than occupied mode"
     annotation (Placement(transformation(extent={{340,40},{360,60}})));
 
```

Comment:
remove the empty line and have the empty line between this section and the equation section.

Reply:
Changed